### ISSUE TITLE
Buying Corps / Army Using Faith Only Produces a Single Unit

### ISSUE THREAD

**Issue by DCTX95** [Azurency/CQUI_Community-Edition#403]
Buying a Corps or Army using Faith through Encampments seems to be resulting in an error. It would only produce a single military unit while spending the correct amount of Faith (of what a Corps / Army would cost.) This would happen to all military units.

**Expected behavior**
A Corps or Army should be produced.

**General information**
 - OS: Windows 10
 - Version: 1.0.0.317
 - Expansion: Rise and Fall + Gathering Storm
 - Other mods: Environment Skin: Sid Meier's Civilization, Colorized Historic Moments

**Comment by Azurency** [comment]
Yes wrong callback in the code. Should be fixed in the latest version.
